app/services/delete_application.rb

```diff
@@ -0,0 +1,90 @@
+class DeleteApplication
+  include ImpersonationAuditHelper
+
+  attr_reader :actor, :application_form, :ticket
+
+  APPLICATION_FORM_FIELDS_TO_REDACT = %i[
+    first_name
+    last_name
+    first_nationality
+    second_nationality
+    english_main_language
+    english_language_details
+    other_language_details
+    date_of_birth
+    further_information
+    phone_number
+    address_line1
+    address_line2
+    address_line3
+    address_line4
+    country
+    postcode
+    disability_disclosure
+    uk_residency_status
+    work_history_explanation
+    becoming_a_teacher
+    subject_knowledge
+    interview_preferences
+    work_history_breaks
+    volunteering_experience
+    equality_and_diversity
+    safeguarding_issues
+    international_address
+    right_to_work_or_study
+    right_to_work_or_study_details
+    third_nationality
+    fourth_nationality
+    fifth_nationality
+    feedback_satisfaction_level
+    feedback_suggestions
+    work_history_status
+  ].freeze
+
+  ASSOCIATIONS_TO_DESTROY = %i[
+    application_work_experiences
+    application_volunteering_experiences
+    application_qualifications
+    application_references
+    application_work_history_breaks
+    application_feedback
+  ].freeze
+
+  def initialize(actor:, application_form:, ticket:)
+    @actor = actor
+    @application_form = application_form
+    @ticket = ticket
```

Comment:
To remove ambiguity, would it make sense for this to be `zendesk_url:` or similar?

Reply:
Left it vague because sometimes we have a Trello reference without the Zendesk one, but it should really be Zendesk. Will update.